mindsdb/interfaces/storage/db.py

```diff
@@ -164,3 +164,8 @@ class Stream(Base):
 
 Base.metadata.create_all(engine)
 orm.configure_mappers()
+metadata = Base.metadata
+
+
+def get_engine():
+    return engine
```

Comment:
unused

Reply:
No, it is in use. It is a part of flask_sqlalchemy.SQLAlchemy instance required interface